pkg/storage/cassandra_dependencies_test.go

```diff
@@ -57,6 +57,21 @@ func TestCassandraCustomTraceTTLParseError(t *testing.T) {
 	assert.Equal(t, "172800", foundValue, "unexpected TRACE_TTL environment var value")
 }
 
+func TestCassandraDefaultPort(t *testing.T) {
+	jaeger := v1.NewJaeger(types.NamespacedName{Name: "my-instance"})
+
+	b := cassandraDeps(jaeger)
+	assert.Len(t, b, 1)
+	assert.Len(t, b[0].Spec.Template.Spec.Containers, 1)
+	foundValue := ""
+	for _, e := range b[0].Spec.Template.Spec.Containers[0].Env {
+		if e.Name == "CQLSH_PORT" {
+			foundValue = e.Value
```

Comment:
Small improvement: you could make an assertion here: `assert.Equal(t, "9042", ...)` and then return. Where your current `assert.Equal` is (at the end of the test), you could have a `assert.Fail(t, "value for CQLSH_PORT not found")`